bundle/manifests/webhook-operator.clusterserviceversion.yaml

```diff
@@ -185,15 +193,11 @@ spec:
       - UPDATE
       resources:
       - webhooktests
-    sideEffects: None
+    sideEffects: null
```

Comment:
And once I manually fix the `admissionReviewVersions` in my bundle, I get a message about this field being invalid as well.

Reply:
This [is fixed](https://github.com/operator-framework/operator-sdk/pull/3903) in version 1.0.1 of the operator SDK.

Reply:
This too :-)